server/api/metrics.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	storeProgressGauge = prometheus.NewGaugeVec(
```

Comment:
Duplicate with `server/metrics.go`?

Reply:
the down api is different from up,  I do not know where pkg I can place

Reply:
How about `server/cluster`?